common/libzkp/impl/src/verifier/euclidv2.rs

```diff
@@ -53,12 +53,13 @@ impl ProofVerifier for EuclidV2Verifier {
     }
 
     fn dump_vk(&self, file: &Path) {
+        use base64::{prelude::BASE64_STANDARD, Engine};
         let f = File::create(file).expect("Failed to open file to dump VK");
 
         let dump = VKDump {
-            chunk_vk: base64::encode(self.chunk_verifier.get_app_vk()),
-            batch_vk: base64::encode(self.batch_verifier.get_app_vk()),
-            bundle_vk: base64::encode(self.bundle_verifier.get_app_vk()),
+            chunk_vk: BASE64_STANDARD.encode(self.chunk_verifier.get_app_vk()),
+            batch_vk: BASE64_STANDARD.encode(self.batch_verifier.get_app_vk()),
+            bundle_vk: BASE64_STANDARD.encode(self.bundle_verifier.get_app_vk()),
```

Comment:
why this change?   base64 upgraded and API changed?

Reply:
yes, new version allow multiple purpose https://docs.rs/base64/latest/base64/prelude/index.html#reexport.BASE64_STANDARD